Replicator/tests/ReplicatorAPITest.cc

```diff
@@ -613,6 +613,215 @@ TEST_CASE_METHOD(ReplicatorAPITest, "Is Document Pending", "[C][Push]") {
 }
 #endif
 
+#ifdef COUCHBASE_ENTERPRISE
```

Comment:
No need to add ifdef for every test, they can be grouped inside (or added to an existing group)